docker-compose.yml

```diff
@@ -104,6 +104,15 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  impersonation:
+    image: lblod/impersonation-service:0.2.0
+    restart: always
+    logging: *default-logging
+    environment:
+      - NODE_ENV=development
+    ports:
+      - "3000:80"
+      - "9222:9229"
```

Comment:
Shouldn't this be in the compose.dev file?

Reply:
absolutely! thx!